lib/flipper/test_help.rb

```diff
@@ -0,0 +1,29 @@
+module Flipper
+  module TestHelp
+    extend self
+
+    def before_all
+      Flipper.configure do |config|
+        config.adapter { Flipper::Adapters::Memory.new }
```

Comment:
Should we do anything different for system tests? I was reminded of [this conversation](https://github.com/flippercloud/flipper/issues/615#issuecomment-1625740455) from a year ago.

Reply:
Yes. I spent some time investigating this today and posted what I found here: https://github.com/flippercloud/flipper/issues/615#issuecomment-1879449641

Now I just need to try it with RSpec, and then figure out how to configure it so it just works.

Might be worth revisiting in a followup PR, or I'm also happy to leave this set until we figure it out.

Reply:
I’m fine with follow up as long as people can override somehow!

Reply:
This broke our system specs, so to answer the question from above:

> Why wait until 2.0? Does it seem like a breaking change?

This was a breaking change :slightly_smiling_face:

Reply:
Not intentionally. Thanks for being helpful.